.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: ruby
 rvm:
-  - 2.2.2
-  - 1.9.3
-  - jruby-19mode
+  - 2.3.7
```

Comment:
No 2.4 build (and I ask only because we're still unable to install 2.5 locally)?

Reply:
Ah gotcha. Added.

Reply:
thanks